app/sim_test.go

```diff
@@ -1,338 +0,0 @@
-package app
-
-import (
```

Comment:
Fair enough to remove the sim tests, which we don't use.

We should probably trim down the circle ci file to match (or maybe you did that already... I think there is a lot of commented out sections there)

Reply:
I did some cleanup in the past already